app/modules/knight.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import annotations
+
+
+class Knight:
+    def __init__(
+            self,
+            name: str = "",
+            power: int = 0,
+            hp: int = 0,
+            armour: list[dict] = [],
+            weapon: dict = {},

```

Comment:
dict and list cannot be default arguments, because they are mutable.
https://www.codecademy.com/learn/learn-intermediate-python-3/modules/int-python-function-arguments/cheatsheet

Reply:
Not fixed.

Reply:
Forgot to git push, sorry about that.